### PR TITLE
Updated all package xml files to version 2

### PR DESCRIPTION
**Pull request by dejaniraai**
Updated all the package.xml files to be of version 2

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
@dejaniraai: looks like github doesn't know about the email address you used to commit your changes.

Would you want to fix that?

**Comment by dejaniraai** [comment]
@gavanderhoorn I am not  sure how to fix the email issue. I do get all the notifications to the Github registered email.

**Comment by gavanderhoorn** [comment]
It's fine. As long as the email used to commit is a correct one.

You can check that [here](https://github.com/ros-industrial/industrial_core/commit/7c800bb8a68c6ecb47b37982ad4415a07ae375e7.patch) at the top (in the `From` line).

File: industrial_msgs/package.xml [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
   <license>BSD</license>
   <url type="website">http://ros.org/wiki/industrial_msg</url>
-  <author>Shaun Edwards</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>std_msgs</build_depend>
-  <build_depend>trajectory_msgs</build_depend>
+
+
   <build_depend>message_generation</build_depend>
```
Comment (trentwall) [inline industrial_msgs/package.xml:18]: forgot to convert <build_depend>message_generation</build_depend> to format 2.

**Review by trentwall — CHANGES_REQUESTED** [review]

File: industrial_msgs/package.xml (hunk 1 above)
Comment (gavanderhoorn) [reply industrial_msgs/package.xml:18]: @trentwall: could you be more specific? What do you believe it should be?

Comment (trentwall) [reply industrial_msgs/package.xml]: <depend>roslaunch</depend>

Comment (trentwall) [reply industrial_msgs/package.xml:18]: <depend>roslaunch</depend>

Comment (gavanderhoorn) [reply industrial_msgs/package.xml:18]: I'm not sure I understand.

Comment (trentwall) [reply industrial_msgs/package.xml:18]: depend roslaunch /depend

Comment (trentwall) [reply industrial_msgs/package.xml:18]: just like the others

Comment (gavanderhoorn) [reply industrial_msgs/package.xml:18]: but `roslaunch` is not the same package as `message_generation`. Package format 2 does not influence that.

Comment (JeremyZoss) [reply industrial_msgs/package.xml:18]: I talked to Trent, and he was confused about the new `<depend>` tag.  No changes are required.  This PR is correct as-is.

**Review by JeremyZoss — APPROVED** [review]
Changes look good.  Builds with no problem.  Thanks for the contribution!

**Pull request merged**